### PR TITLE
don't return or accept sms_sender in update service anymore

### PR DESCRIPTION
**Pull request by leohemsted**
it's no longer a field on the service

### CODE REVIEW

**Comment by venusbb** [comment]
Delete here in admin and check if there are other references
https://github.com/alphagov/notifications-admin/blob/master/app/notify_client/service_api_client.py#L99

File: app/schemas.py [hunk 1]
```diff
@@ -216,9 +214,6 @@ def service_permissions(self, service):
     def get_reply_to_email_address(self, service):
```
Comment (servingUpAces) [inline app/schemas.py]: Does the same apply to reply_to_email?

File: app/schemas.py [hunk 2]
```diff
@@ -216,9 +214,6 @@ def service_permissions(self, service):
...
         return service.get_default_reply_to_email_address()
 
-    def get_sms_sender(self, service):
-        return service.get_default_sms_sender()
-
     def get_letter_contact(self, service):
```
Comment (servingUpAces) [inline app/schemas.py]: Does the same apply to letter_contact?

Comment (leohemsted) [reply app/schemas.py]: used in admin here - https://github.com/alphagov/notifications-admin/blob/master/app/main/views/service_settings.py#L663

File: app/schemas.py (hunk 1 above)
Comment (leohemsted) [reply app/schemas.py]: yup you're right - removed

**Pull request merged**